README.md

```diff
@@ -70,13 +101,54 @@ $ ctest
 ```
 
 ## Installing OpenSMT2
-As long as you haven't disabled building them, the path to the OpenSMT executable is `<BUILD_DIR>/src/bin/opensmt`, the OpenSMT libraries are located in `<BUILD_DIR>/src/api`.
-To install OpenSMT in your system simply run
+[//]: # "the binary should be placed into `<build_dir>/bin/opensmt`"
+As long as you haven't disabled building them, the path to the OpenSMT executable is `<build_dir>/opensmt`, the OpenSMT libraries are located in `<build_dir>/lib`.
+
+To install OpenSMT into your system simply run:
 ```
 $ make install
 ```
-The install directory can be customized using cmake variable CMAKE_INSTALL_PREFIX. The default is `/usr/local`.
-This installs the library in the folder `<INSTALL_DIR>/lib` and puts the necessary header files in the folder `<INSTALL_DIR>/include/opensmt`.
+(or `make install-release`).
+This runs `cmake --install <build_dir>`.
+The install directory can be customized using option `INSTALL_DIR=<install_dir>`.
+The default is `/usr/local`.
+In such a case, the command above may be neccesary to run with `sudo`.
+The default build to be installed is Release. To install the Debug build instead, use `make install-debug`.
+
+The option `INSTALL_DIR` may used with all `make [release|debug]` and `make install[-release|-debug]`.
+In the case of the build rules, the `<install_dir>` is configured within the build directory and is used each time `make install[-release|-debug]` is used without the additional option `INSTALL_DIR`.
+In the case of the install rules, the `<install_dir>` overrides the previously configured one.
+
+Option `CMAKE_INSTALL_FLAGS` may be used for any additional arguments to be passed to `cmake --install <build_dir>`.
+
+The command installs the executable binary into the folder `<install_dir>/bin`, the library into the folder `<install_dir>/lib` and puts the necessary header files into the folder `<install_dir>/include/opensmt`.
+
+### Examples
+
+Build and install with the default values (assuming that writing into `/usr/local` directory requires root priviledges):
+```
+make && sudo make install
+```
+
+Build Release and install into a local directory `local_dir`:
+```
+make release INSTALL_DIR=local_dir && make install-release
+```
+
+Build Debug and install it into a local directory `local_dir-debug`:
+```
+make debug INSTALL_DIR=local_dir-debug && make install-debug
+```
+
+Build (and configure) with the default values but install into `local_dir`:
+```
+make && make install INSTALL_DIR=local_dir
+```
+
+Build and configure to by default install into `local_dir` but also install into `local_dir2`:
```

Comment:
I am not sure how to parse this sentence.

Reply:
Another try:
Build and configure to install into `local_dir` and install there, but also perform another installation into `local_dir2`